python/pyarrow/tests/util.py

```diff
@@ -30,6 +30,8 @@
 import string
 import subprocess
 import sys
+import re
```

Comment:
Nit, but please let's keep stdlib imports in alphabetical order.

Reply:
Made the change.